Kudu.Services/Deployment/PushDeploymentController.cs

```diff
@@ -138,6 +140,185 @@ public class PushDeploymentController : ApiController
             }
         }
 
+        [HttpPost]
+        [HttpPut]
+        public async Task<HttpResponseMessage> OnePushDeploy(
+            [FromUri] bool isAsync = false,
+            [FromUri] string author = null,
+            [FromUri] string authorEmail = null,
+            [FromUri] string deployer = OneDeploy,
+            [FromUri] string message = DefaultMessage)
+        {
+            using (_tracer.Step("OnePushDeploy"))
+            {
+                var response = Request.CreateResponse();
+
+                //Parses all query parameters into queryCollection
+                var queryCollection = Request.RequestUri.ParseQueryString();
+                var deployType = queryCollection[Constants.Type];
+                var website_framework = System.Environment.GetEnvironmentVariable(Constants.Framework).ToLower();
+                //Checks the "path" queary parameter, path root at /wwwroot TBD if need to change to /home
+                string path = queryCollection[Constants.FilePath];
```

Comment:
>string [](start = 16, length = 6)

var for sake of consistency. Or if you prefer to use the type explicitly, do the same elsewhere.